db/migrate/20240529025722_create_ships.rb

```diff
@@ -0,0 +1,14 @@
+class CreateShips < ActiveRecord::Migration[7.1]
+  def change
+    create_table :ships do |t|
+      t.string :name
+      t.text :description
+      t.decimal :price_per_day
+      t.date :available_from
+      t.date :available_to
+      t.references :user, null: false, foreign_key: true
+
```

Comment:
Missing :category attribute, we should also yet define what categories of ships will exist in order to define it as a collection

Reply:
We should also add a :capacity attribute, it feels important to inform